### DISCUSSION TITLE
Error while deploy my nextjs app on vercel

### DISCUSSION THREAD

**Discussion by igeral** [vercel/next.js#55119] [Help]
### Summary

I am trying to deploy my nextjs app on vercel. The app runs on my local machine but when deploying I get the following error

### Additional information

```js
./node_modules/keen-slider/keen-slider.min.css
Module build failed: Error: Final loader (./node_modules/next/dist/build/webpack/loaders/error-loader.js) didn't return a Buffer or String
    at processResult (/vercel/path0/node_modules/next/dist/compiled/webpack/bundle5.js:28:398849)
    at /vercel/path0/node_modules/next/dist/compiled/webpack/bundle5.js:28:400319
    at /vercel/path0/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:8727
    at iterateNormalLoaders (/vercel/path0/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5565)
    at iterateNormalLoaders (/vercel/path0/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5650)
    at /vercel/path0/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5879
    at runSyncOrAsync (/vercel/path0/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:4185)
    at iterateNormalLoaders (/vercel/path0/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5782)
    at /vercel/path0/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5426
    at /vercel/path0/node_modules/next/dist/compiled/webpack/bundle5.js:28:399794
```

**Comment by icyJoseph** [comment]
Does it do the same if you build it locally? With `next build` and `next start` -

Looks like somehow your webpack config is failing to load the CSS file? Do you have any custom changes to your webpack config?

Also make sure you have the same Node.js versions in Vercel, and locally. Check for syntax errors in your code as well.

**Reply by igeral** [reply]
The error came from a library I was using in my project. I have Uninstalled the library and used and alternate library. 
Thanks.

**Comment by OkiljonDadakhanov** [comment]
![image](https://github.com/user-attachments/assets/b87c437b-ce86-4da0-a259-a388c63c18a6)
I have this error unsolved for 3 days, I did everything that needs to be done but no change. I deleted node modules, package json and reinstalled them, no change.
